cmd/version-main.go

```diff
@@ -44,13 +44,6 @@ func mainVersion(ctx *cli.Context) {
 		cli.ShowCommandHelpAndExit(ctx, "version", 1)
 	}
 
-	// Initialization routine, such as config loading, enable logging, ..
-	minioInit(ctx)
```

Comment:
One more thing missing here is that upon `--help` it shouldn't list `--quiet` flag at all for this command alone and also `--quiet` should move to `server` command only flag not a global flag.